appendices/spirvenv.txt

```diff
@@ -614,6 +614,10 @@ ifndef::VK_VERSION_1_3,VK_KHR_format_feature_flags2[]
     operand of 2 and an "`Image Format`" operand of code:Unknown must: be
     decorated with code:NonReadable.
 endif::VK_VERSION_1_3,VK_KHR_format_feature_flags2[]
+  * code:OpImageWrite to any variable created with a code:Type of
+    code:OpTypeImage must: have the code:Texel operand contain at least
+    as many components as the code:Image code:Format according to the
+    <<spirvenv-image-formats,SPIR-V Image Format compatibility table>>
```

Comment:
```suggestion
  * code:OpImageWrite to any code:Image whose code:Image code:Format is not
    code:Unknown must: have the code:Texel operand contain at least
    as many components as the corresponding elink:VkFormat as given in the
    <<spirvenv-image-formats,SPIR-V Image Format compatibility table>>
```

It's a bit wordier than I would like, but I think it's unambiguous. I removed the "variable with type code:OpTypeImage" because that is required for all writes. (I think that's left over from the language about Sampled operand being 2)

Reply:
This looks way better! Thanks as always for time/energy in helping rewording these "fun" VUs